rest/persistent_config_test.go

```diff
@@ -1670,9 +1635,38 @@ func TestPersistentConfigNoBucketField(t *testing.T) {
 }
 
 // startBootstrapServerWithoutConfigPolling starts a server with config polling disabled, and returns the server context.
-func startBootstrapServerWithoutConfigPolling(t *testing.T) (*ServerContext, func()) {
+func startBootstrapServerWithoutConfigPolling(t *testing.T, useXattrConfig bool) (*ServerContext, func()) {
 	config := BootstrapStartupConfigForTest(t)
 	// "disable" config polling for this test, to avoid non-deterministic test output based on polling times
 	config.Bootstrap.ConfigUpdateFrequency = base.NewConfigDuration(time.Hour * 24)
+	config.Unsupported.UseXattrConfig = base.BoolPtr(useXattrConfig)
 	return StartServerWithConfig(t, &config)
 }
+
+type persistentConfigTestCase struct {
+	name        string
+	xattrConfig bool
+}
+
+func persistentConfigTestCases() []persistentConfigTestCase {
+
+	if base.UnitTestUrlIsWalrus() {
```

Comment:
rosmar doesn't exist on 3.1 at all and we don't have bootstrap support? I'd probably remove this from here, or indicate it here.

This is fine because all of these tests are already skipped without Server.